.github/workflows/performance.yml

```diff
@@ -67,12 +62,19 @@ jobs:
 
       - name: WordPress Docker container debug information
         run: |
+          docker -v
+          docker-compose -v
           docker-compose run --rm mysql mysql --version
           docker-compose run --rm php php --version
           docker-compose run --rm php php -m
           docker-compose run --rm php php -i
           docker-compose run --rm php locale -a
 
+      - name: Install MU plugin
+        run: |
+          mkdir ./src/wp-content/mu-plugins
+          cp ./tests/performance/plugins/server-timing/mu-plugin.php ./src/wp-content/mu-plugins/server-timing.php
```

Comment:
Nit-pick: It would make more sense to me to have this file in `./tests/performance/mu-plugins/server-timing.php`, as that would indicate better where it should go. Seems unnecessary to use a different structure in the source directory for this MU plugin.

Reply:
Thanks @felixarntz for the review. https://github.com/10up/wordpress-develop/pull/7/commits/85a5c96fe2bbccc806c4b8eb06863a26c4285617 updates the MU plugin path.